### DISCUSSION TITLE
templaterCreateNote does not seem to work

### DISCUSSION THREAD

**Discussion by gillesbonnet** [mProjectsCode/obsidian-meta-bind-plugin#220] [Q&A] [answered]
Templater buttons do not seem to work for me. I did not succeed to make the [example](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/guides/buttons/#templater-create-note-action) from the doc work. 
Also the example from the [exampleVault](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/tree/master/exampleVault) does not work for me: I downloaded the exampleVault from the Github repository of the project, open it, installed and activated the plugins "Meta Bind" and Templater. Then, in the note "Button Example" there is the button

```meta-bind-button
label: Templater
hidden: false
id: ""
style: default
actions:
  - type: templaterCreateNote
    templateFile: "templates/templater/Templater Template.md"
    fileName: Button Templater Test

```

When I click on it, I receives the popup message: "metaBind | Error while running button action. Check the console for details."
In the console I see this:
<img width="451" alt="image" src="https://github.com/mProjectsCode/obsidian-meta-bind-plugin/assets/14077290/1c400ded-c060-427c-bd75-89c187cebd6c">

Does someone has a suggestion to try something I might have forgotten?

**Comment by mProjectsCode** [comment]
Oh, I seem to have messed something up in the latest update.

**Reply by mProjectsCode** [reply, marked as answer]
Fixed, see #221

**Reply by gillesbonnet** [reply]
Thank you so much for your great reactivity (and an awesome plugin).
